game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -481,10 +480,7 @@ public void windowGainedFocus(final WindowEvent e) {
     tooltipManager = new MapUnitTooltipManager(mapPanel);
     mapPanel.addMapSelectionListener(mapSelectionListener);
     mapPanel.addMouseOverUnitListener(
-        (units, territory) -> {
-          unitsBeingMousedOver = units;
-          tooltipManager.updateTooltip(getUnitInfo());
-        });
+        (units, territory) -> tooltipManager.updateTooltip(getUnitInfo()));
```

Comment:
Why not just pass units to `getUnitInfo()`?
This way there's no need to expose a getter, or am I missing something?

Reply:
I did give that a shot, the second usage of `getUnitInfo` IIRC does not work well with it. If it did work, then perhaps the changes then became just extremely involved and would not have been reviewable. Perhaps after this PR it can be done more cleanly, or the original problem in the second usage would become clear.

Reply:
Interesting, weird nevertheless.